modules/tinymce/src/plugins/codesample/test/ts/module/CodeSampleTestUtils.ts

```diff
@@ -19,8 +19,8 @@ const setTextareaContent = (content: string): void => {
 };
 
 const assertCodeSampleDialog = (expectedLanguage: string, expectedContent: string): void => {
-  const select = document.querySelector('div[role="dialog"] select') as HTMLSelectElement;
-  assert.equal(select.value, expectedLanguage, 'Asserting language dropdown is ' + expectedLanguage);
+  const select = document.querySelector('div[role="dialog"] div[class="tox-listboxfield"] button') as HTMLSelectElement;
```

Comment:
```suggestion
  const select = document.querySelector('div[role="dialog"] div[class="tox-listboxfield"] button') as HTMLButtonElement;
```

Might have missed this